README.md

```diff
@@ -7,6 +7,14 @@
 
 A NodeJS API-based framework. The framework is at its initial state and development is currently being worked on. Therefore, expect to see incomplete work.
 
+
+# Philosophy
```

Comment:
Good work adding the philosophy.
We would appreciate more contribution in the future.